projects/uswds-components/src/lib/textarea/textarea.component.ts

```diff
@@ -0,0 +1,175 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  Optional,
+  Output,
+  Self,
+} from '@angular/core';
+import { ControlValueAccessor, NgControl } from '@angular/forms';
+import { FormFieldComponent } from '../form-field/form-field.component';
+import { Key } from '../util/key';
+
+let nextId = 0;
+
+@Component({
+  selector: 'usa-textarea',
+  templateUrl: './textarea.component.html',
+})
+export class UsaTextareaComponent implements ControlValueAccessor {
+  model: string = '';
+
+  private _onChange = (_: any) => {};
+  private _onTouched = () => {};
+  /**
+   * Sets the ariaLabel attribute
+   */
+  @Input() ariaLabel = 'Textarea component';
+  /**
+   * Sets the rowHeight
+   */
+  @Input() rowHeight: number;
+  /**
+   * Sets the maxLength attribute
+   */
+  @Input() maxlength: number;
+
+  /**
+   * Sets the maxLength attribute
+   */
+  @Input() minlength: number;
+
+  /**
+   * Sets the id attribute
+   */
+  @Input() id = `usa-textarea-${nextId++}`;
+  /**
+   * Sets the placeholder attribute
+   */
+  @Input() placeholder = '';
+  /**
+   * Sets the name attribute
+   */
+  @Input() name = 'textarea';
+
+  /**
+   * Optional assistance text to be set when placeholder attribute is used
+   */
+  @Input() title: string;
+
+  /**
+   * Sets the disabled attribute
+   */
+  @Input() disabled: boolean;
+
+  /**
+   * Emits focus event
+   */
+  @Output() onBlur: EventEmitter<string> = new EventEmitter(null);
+
+  /**
+   * deprecated, emits value change events
```

Comment:
Normally with deprecated pieces, it's also helpful for users to know what they should use instead. Could you update the comment to also recommend what the user should use to listen for value changes?

Reply:
Updated comments